src/Shared/DaprDefaults.cs

```diff
@@ -19,23 +19,33 @@ internal static class DaprDefaults
     {
         private static string httpEndpoint;
         private static string grpcEndpoint;
-        private static string apiToken;
+        private static string daprApiToken;
+        private static string appApiToken;
 
-        public static string GetDefaultApiToken()
+        public static string GetDefaultDaprApiToken()
```

Comment:
If you want to do this, I'd recommend keeping the old method and having it point to this. With this change, you'll break anyone who references this method at compile time, which isn't a good experience.

Reply:
Yep, shall we keep the old method with
```c#
[Obsolete("Use GetDefaultDaprApiToken method.")]
```
?

Since this attribute can also remind APP developers who use the Dapr API token incorrectly.

Reply:
@JadynWong - I just spoke with @rynowak who explained to me that since this is an internal class, removing the method won't actually break anyone. Sorry to make you redo work that I called out but you're free to delete this method and use the better named one.

Reply:
I have reverted it and add comments.